bin/entrypoint

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+set -e
+
+export EXTRA_KUBECTL_VERSIONS=$(echo $(list-extra-kubectl-versions))
```

Comment:
Is echo needed?

```suggestion
export EXTRA_KUBECTL_VERSIONS="$(list-extra-kubectl-versions)"
```

Reply:
@fsouza so the idea here was to ensure the value was a single line to reduce the complexity within the go module.. 

![Screen Shot 2019-07-19 at 11 19 17 AM](https://user-images.githubusercontent.com/639255/61546308-4a5ba300-aa17-11e9-8001-bd834ea1067c.png)

a multi-line value would have required a bit more logic around the following:

https://github.com/nytimes/drone-gke/blob/2123082469d22caff6205e0ab4f8ded0ae3de306/main.go#L45

Reply:
I would add a comment above to add the reason why